src/apis/address.ts

```
@@ -0,0 +1,73 @@
+}
+
+export const getAddress = async () => {
+  try {
+    const response = await instance.get(`${address}`, {
+      headers: {
+        Authorization: `Bearer ${await getItem('accessToken')}`
+      }
+    });
+    return response;
+  } catch (err) {
+    throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)